### PR TITLE
Update Day 2 UI docs with recent changes (SCRD-8484)

### PR DESCRIPTION
**Pull request by jeremy-moffitt**
Updated content and screenshots for Day2 UI based on recent changes:

* Server details includes network and filesystem utilization information
* Renamed servers tab in topology
* Ability to manually add NIC mapping in server management dialogs
* added "deactivate" server feature details
* added "activate" server feature details
* added "delete" server feature details
* updated options during replace/add server

### CODE REVIEW

**Comment by jeremy-moffitt** [comment]
for non-doc-writer reviewers:
- clone this repo and check out the branch
- install docker if not already installed
- clone https://github.com/openSUSE/daps2docker
- from daps2docker repo, run `./daps2docker.sh <path-to-doc-repo>/DC-suse-openstack-cloud-operations`  , for example on my system this is `sudo ./daps2docker.sh ~/upstream/doc-cloud/DC-suse-openstack-cloud-operations`

**Comment by jeremy-moffitt** [comment]
Going to add a section on the "Model" tab, which was not complete when SCRD-8484 was first created, will have a new patchset later today

**Comment by jeremy-moffitt** [comment]
updated to add configuration and model tab documentation

File: xml/clm_admin_addserver.xml [hunk 1]
```diff
@@ -332,4 +387,189 @@
...
+   <imagedata fileref="clm_admin_activate_2.png" width="75%" format="PNG"/>
+  </imageobject>
+  <imageobject role="html">
+   <imagedata fileref="clm_admin_activate_2.png"/>
+  </imageobject>
+ </mediaobject>
+ </figure>
+</section>
+<section xml:id="clm-admin-deactivate-server">
+ <title>Dectivating Servers</title>
```
Comment (csymons-suse) [inline xml/clm_admin_addserver.xml]: Deactivating

File: xml/clm_admin_addserver.xml [hunk 2]
```diff
@@ -332,4 +387,189 @@
...
+  <imageobject role="fo">
+   <imagedata fileref="clm_admin_delete_2.png" width="75%" format="PNG"/>
+  </imageobject>
+  <imageobject role="html">
+   <imagedata fileref="clm_admin_delete_2.png"/>
+  </imageobject>
+ </mediaobject>
+ </figure>
+ <para>
+  If the &compnode; is not reachable (ssh from the deployer is not
```
Comment (csymons-suse) [inline xml/clm_admin_addserver.xml]: ssh > SSH upper case

File: xml/clm_admin_addserver.xml [hunk 3]
```diff
@@ -332,4 +387,189 @@
...
+  possible), a warning will appear, requesting confirmation that
+  the node is shutdown or otherwise removed from the environment.
+  Reachable &compnode;s will be shutdown as part of the deletion
```
Comment (csymons-suse) [inline xml/clm_admin_addserver.xml]: shutdown > shut down

File: xml/day2_ui_documentation.xml [hunk 4]
```diff
@@ -158,6 +158,86 @@
   </mediaobject>
  </figure>
 </section>
+<section xml:id="admin_ui_suse_cloud_configuration">
+ <title>Configuration</title>
+ <para>
+  The Configuration tab displays services that are deployed in the cloud and
+  the configuration files associated with those services. Services
+  may be reconfigured by editing the `.j2` files listed and clicking the
+  "Update" button.
```
Comment (csymons-suse) [inline xml/day2_ui_documentation.xml]: <filename>.j2</filename>
<guimenu>Update</guimenu> button.

File: xml/day2_ui_documentation.xml [hunk 5]
```diff
@@ -158,6 +158,86 @@
...
+   <imageobject role="fo">
+    <imagedata fileref="clm_admin_services_configuration_1.png" width="75%"/>
+   </imageobject>
+   <imageobject role="html">
+    <imagedata fileref="clm_admin_services_configuration_1.png"/>
+   </imageobject>
+  </mediaobject>
+ </figure>
+ <para>
+  After editing the service configuration, click the "Update" button
```
Comment (csymons-suse) [inline xml/day2_ui_documentation.xml]: <guimenu>Update</guimenu> button

File: xml/day2_ui_documentation.xml [hunk 6]
```diff
@@ -158,6 +158,86 @@
...
+   <imageobject role="html">
+    <imagedata fileref="clm_admin_services_configuration_2.png"/>
+   </imageobject>
+  </mediaobject>
+ </figure>
+</section>
+<section xml:id="admin_ui_suse_cloud_model">
+ <title>Model</title>
+ <para>
+  The Model tab displays the yml files that make up the primary configuration
```
Comment (csymons-suse) [inline xml/day2_ui_documentation.xml]: The <guimenu>Model</guimenu> tab

File: xml/day2_ui_documentation.xml (hunk 4 above)
Comment (csymons-suse) [reply xml/day2_ui_documentation.xml]: use filename tags around .j2
use guimenu tags around button and tab names

File: xml/day2_ui_documentation.xml (hunk 5 above)
Comment (csymons-suse) [reply xml/day2_ui_documentation.xml]: see previous tagging suggestions

File: xml/day2_ui_documentation.xml (hunk 6 above)
Comment (csymons-suse) [reply xml/day2_ui_documentation.xml]: filename-tagModelclose-filename-tag

File: xml/day2_ui_documentation.xml [hunk 7]
```diff
@@ -158,6 +158,86 @@
...
+  of the Cloud. Model modifications can be validated here for basic syntax
+  but are not deployed from this page. The model tab allows users to make
+  advanced updates to their datamodel when a server addition or replacement
+  fails and requires manual modification of the model to continue or roll-back
+  changes.
+ </para>
+ <para>
+  Clicking "validate" will commit the model changes, run the configuration
```
Comment (csymons-suse) [inline xml/day2_ui_documentation.xml]: guimenu tags around validate

File: xml/day2_ui_documentation.xml (hunk 6 above)
Comment (jeremy-moffitt) [reply xml/day2_ui_documentation.xml]: I'm a little fuzzy on this one, are tabs usually wrapped as files?

Comment (jeremy-moffitt) [reply xml/day2_ui_documentation.xml]: I see the other tab mentions as guimenu, I'll try using that, let me know if thats not right in this spot

File: xml/clm_admin_addserver.xml (hunk 1 above)
Comment (jeremy-moffitt) [reply xml/clm_admin_addserver.xml]: fixed in latest version

File: xml/clm_admin_addserver.xml (hunk 3 above)
Comment (jeremy-moffitt) [reply xml/clm_admin_addserver.xml]: fixed in latest version

File: xml/day2_ui_documentation.xml (hunk 5 above)
Comment (jeremy-moffitt) [reply xml/day2_ui_documentation.xml]: fixed in latest version

File: xml/day2_ui_documentation.xml (hunk 7 above)
Comment (jeremy-moffitt) [reply xml/day2_ui_documentation.xml]: fixed in latest version

File: xml/day2_ui_documentation.xml (hunk 6 above)
Comment (csymons-suse) [reply xml/day2_ui_documentation.xml]: I was wrong. `guimenu` it is
files and directories are wrapped with `filename`

**Pull request merged**